components/script/dom/htmlinputelement.rs

```diff
@@ -176,7 +176,7 @@ impl HTMLInputElement {
             .map_or_else(|| atom!(""), |a| a.value().as_atom().to_owned())
     }
 
-    // https://html.spec.whatwg.org/multipage/#input-type-attr-summary
+    // https://html.spec.whatwg.org/multipage/#common-input-element-apis
```

Comment:
Why?

Reply:
Because the old link doesn't point to somewhere that can explain value modes.

Reply:
https://html.spec.whatwg.org/multipage/input.html#dom-input-value is a better link, then.